src/test/interface/io/json/json_parser_test.cpp

```diff
@@ -363,231 +363,187 @@ TEST(ioJson,jsonParserErr01) {
 
 TEST(ioJson,jsonParserErr02) {
   test_exception("[ .5 ]",
-                 "illegal value, expecting object, array, number, string, or literal true/false/null\n");
+                 "Invalid value.\n");
 }
 
 TEST(ioJson,jsonParserErr02a) {
   test_exception("[ 0",
-                 "unexpected end of stream\n");
+                 "Missing a comma or ']' after an array element.\n");
 }
 
 TEST(ioJson,jsonParserErr02b) {
   test_exception("[ 0.",
-                 "unexpected end of stream\n");
+                 "Miss fraction part in number.\n");
 }
 
 
 TEST(ioJson,jsonParserErr02c) {
   test_exception("[ 99.9",
-                 "unexpected end of stream\n");
+                 "Missing a comma or ']' after an array element.\n");
 }
 
 
 TEST(ioJson,jsonParserErr03) {
   test_exception("[ 000.005 ]",
-                 "zero padded numbers not allowed\n");
+                 "Missing a comma or ']' after an array element.\n");
 }
 
 TEST(ioJson,jsonParserErr04) {
   test_exception("[ 1. ]",
-                 "expected digit after decimal\n");
+                 "Miss fraction part in number.\n");
```

Comment:
should be "missing" - again, if this under our control?

Reply:
Yes, its easy to change, will do that.